### PR TITLE
Update to Cosmos/eb8ce1647

### PR DESCRIPTION
**Pull request by o1iver**
Updating to https://github.com/Actyx/Cosmos/commit/eb8ce1647

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,7 @@ The code can be found in `src/dashboard`.
 These three apps together form an imaginary, yet realistic, factory solution on the Actyx platform.
 
 ![factory solution](./images/advanced-tutorial-nodes.svg)
+
+### What are those weird start/end comments?
```
Comment (wwerner) [inline README.md:29]: They are not in the advanced example (only in the academy project), so this section can be removed.

File: src/dashboard/root.tsx [hunk 2]
```diff
@@ -4,15 +4,36 @@ import * as ReactDOM from 'react-dom'
...
 import { App } from './App'
 
-const onError = () => {
+let error: unknown = undefined
+const onError = (e: unknown) => {
+  error = e
   setTimeout(() => location.reload(), 2500)
 }
 
+function Loading() {
```
Comment (wwerner) [inline src/dashboard/root.tsx:13]: I'd probably make the `Loading` component very simple and remove the timing. It's nice, but people will implement their own loading component anyway.

The important thing is the reload on error.

**Review by wwerner — CHANGES_REQUESTED** [review]
See comments.
* The 'weird comment'  note can be removed
* I'd keep reloading in the error handler
* I'd make the `loading` component simpler

File: src/dashboard/root.tsx (hunk 2 above)
Comment (o1iver) [reply src/dashboard/root.tsx:13]: advanced-tutorial

I just tested it. Works fine, so I will merge this for now. Feel free to change this anytime though.

File: README.md (hunk 1 above)
Comment (o1iver) [reply README.md:29]: Removing

**Comment by o1iver** [comment]
Changing the README as commented by @wwerner. Will open a new PR.

**Pull request closed without merging**